driver-mct/main/seq_diag_mct.F90

```diff
@@ -1252,15 +1315,17 @@ end subroutine seq_diag_rof_mct
   !     Compute global glc input/output flux diagnostics
   !
   ! !REVISION HISTORY:
-  !    2008-jul-10 - T. Craig - update
+  !    2024-Sept. - S. Price - update
```

Comment:
You should add a line for your update instead of replacing Tony's.

Reply:
Done.